ai-service/src/ai_service/main.py

```
@@ -1,53 +1,91 @@
-    db.add_chunks([code], [embedding])
+load_dotenv()
+from ai_service import (
+    db,
+    embedder,
+    ollama_client,
+    errors,
+    project_ingestor,
+)
+
+
+def ingest_github_project(repo_url):
```

Ruff (0.12.2)

Missing return type annotation for public function `ingest_github_project`

Add return type annotation: `None`

(ANN201)